configs/taco/IA.yaml

```diff
@@ -5,7 +5,7 @@ links:
   url: https://coronavirus.iowa.gov/
   overseerScript: |
     page.manualWait();
-    await page.waitForDelay(30000);
+    await page.waitForDelay(60000);
```

Comment:
The overall screenshotter logic by default times out after 60 sec so we should add this to the affected configs to increase it (e.g. see the primary config in https://github.com/COVID19Tracking/screenshots/blob/main/configs/taco/IN.yaml):

```
  requestSettings: 
    maxWait: 60000
```

Reply:
added--sorry, missed this comment before!